README.md

```diff
@@ -17,17 +17,66 @@ A modern age calculator Android app built with Python and Flet Framework featuri
 - GitHub integration button
 - Responsive layout
 
-## 📦 Devlopment Installation
+## 📦 Devlopment Instructions
+
+Follow these steps to set up and run the Age Calculator application locally on your machine .
+
+### ✅ Prerequisites
+- Python **3.9+** installed
+- Recommended: Use a **virtual environment (venv)** to manage project dependencies
+
+### ⚙️ Step 1: Set up the Environment
+
+Create Virtual Eenvirnoment:
+```bash
+python -m venv .venv
+```
```

Comment:
Clone Repo should be the first step.
How someone will setup the venv without clone the repo.